pkg/sample/sample.go

```diff
@@ -0,0 +1,12 @@
+package sample
+
+// Add takes two integer inputs and returns the sum of the two inputs
+func Add(a int, b int) int {
```

Comment:
Are we doing this just to have something? Can we include something more meaningful?

Reply:
Just to have something right now, placeholders till we progress this epic.